src/Puzzle.js

```diff
@@ -1,31 +1,23 @@
 // import Wheel from "./Wheel.js";
 // import data from "./data.js";
 import Game from "./Game.js";
-// import Round from "./Round.js";
-// import Player from "./Player.js";
+import Round from "./Round.js";
+import Player from "./Player.js";
 // import domUpdates from "./domUpdates.js";
 
 class Puzzle {
-  constructor() {
+  constructor(puzzleArry) {
 
   this.hint = null;
-  this.puzzleobj = null;
+  this.puzzleArray = [];
+  this.puzzleobj = {};
```

Comment:
My understanding was that it's not considered best practice to use data types in variable names.  At some point can we double check with an instructor on that?